addons/core/functions/server/fnc_missingModMessage.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+/*
+  Name: TFAR_fnc_missingModMessage
+
+  Author: NKey, Garth de Wet (L-H), Dorbedo
+    Returns a message on Clientside about missing TFAR
+
+  Arguments:
+    None
+
+  Return Value:
+    None
+
+  Example:
+    call TFAR_fnc_missingModMessage;
+
+  Public: No
+*/
+
+if(isServer) exitWith {};
+waitUntil {sleep 0.1;time > 3};
+if !(isClass(configFile >> "CfgPatches" >> "tfar_core")) exitWith {
+    [player, format["%1: LOOKS LIKE TASK FORCE RADIO ADDON IS NOT ENABLED OR VERSION LESS THAN 1.0", name player]] remoteExec ["globalChat", -2];
+    if (isNull (uiNamespace getVariable ["BIS_fnc_arsenal_cam", objNull])) then {
+        ["LOOKS LIKE TASK FORCE RADIO ADDON IS NOT ENABLED OR VERSION LESS THAN 1.0"] call "BIS_fnc_guiMessage";
```

Comment:
since when does call support a string right handed?

Reply:
it doesn't